great_expectations/datasource/generator/glob_reader_generator.py

```diff
@@ -156,11 +164,11 @@ def _get_iterator(self, generator_asset, reader_options=None, limit=None):
         paths = glob.glob(os.path.join(self.base_directory, glob_config["glob"]))
         return self._build_batch_kwargs_path_iter(paths, glob_config, reader_options=reader_options, limit=limit)
 
-    def _build_batch_kwargs_path_iter(self, path_list, glob_config, reader_options=None, limit=None):
+    def _build_batch_kwargs_path_iter(self, path_list, glob_config, reader_options=None, limit=None, reader_method=None):
         for path in path_list:
-            yield self._build_batch_kwargs_from_path(path, glob_config, reader_options=reader_options, limit=limit)
+            yield self._build_batch_kwargs_from_path(path, glob_config, reader_options=reader_options, limit=limit, reader_method=None)
```

Comment:
I think between `self.reader_method` and `glob_config`, you've already got the relevant info here.
```suggestion
            yield self._build_batch_kwargs_from_path(path, glob_config, reader_options=reader_options, limit=limit)
```